library/alloc/src/bstr.rs

```diff
@@ -626,6 +652,7 @@ impl<'a> From<&'a ByteStr> for Cow<'a, ByteStr> {
 #[cfg(not(test))] // https://github.com/rust-lang/rust/issues/135100
 #[unstable(feature = "bstr", issue = "134915")]
 impl From<Box<[u8]>> for Box<ByteStr> {
+    /// Create a `Box<[u8]>` from `Box<ByteStr>`s raw.
```

Comment:
This doc comment makes no sense to me, I had to look at the source to realize what you meant.

Were some of these doc comments generated? Or do you have patterns you established and copied them around?